exercises/Euclidean/python/tests.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from main import *
+import math
+import random
+
+class Test(unittest.TestCase):
```

Comment:
Would be good to also test edge cases (like if one of the inputs is 1), and to test the retrieval of user input acts as you expect (using mocking https://stackoverflow.com/questions/47690020/python-3-unit-tests-with-user-input)